### ISSUE TITLE
[REFACTOR] Replace DifferenceKit with Swift 5.1 collection.difference(from:)

### ISSUE THREAD

**Issue by cardoso** [cardoso/ReduxMovieDB#28]

**Comment by lareenmelo** [comment]
Hi! May I start implementing this?

**Comment by cardoso** [comment]
@lareenmelo of course! Thanks!

**Comment by akarsh** [comment]
I don't think this is possible since difference(from:) is available from iOS 13. The refactor also requires the implementation of tableView reload and do an animation similar to the current reload animation provided by differenceKit.

**Comment by cardoso** [comment]
@akarsh thanks for the note. No problem with iOS 13. The deployment target can be raised.
